docs/tutorial/tutorial-5/index.rst

```diff
@@ -32,15 +32,6 @@ So far, we've been running and testing our application on the desktop. However,
 BeeWare also supports mobile platforms - and the application we've written
 can be deployed to your mobile device, too!
 
-.. admonition:: iOS only... for now.
-
-    The BeeWare project *has* tooling for Android. Unfortunately, due to some
-    recent changes in the Android ecosystem, combined with some changes in Toga
-    itself, means that our Android tooling is currently broken. We're working
-    on it, and hope to have a solution in the very near future.
-
-    In the meantime, this tutorial only works on iOS.
-
```

Comment:
Mental note: There's a similar warning on the homepage. We should revisit that text, too.

Reply:
I made a follow-up PR to discuss that: https://github.com/beeware/beeware/pull/43